pkg/cluster/controllers/database/utils.go

```diff
@@ -141,6 +144,32 @@ func (p *PostgreSQLController) GetPostgreVersion() string {
 	return p.HarborCluster.Spec.InClusterDatabase.PostgresSQLSpec.Version
 }
 
+func (p *PostgreSQLController) GetPostgreParameters() map[string]string {
+	return map[string]string{
+		"max_connections": p.GetPosgresMaxConnections(),
+	}
+}
+
+func (p *PostgreSQLController) GetPosgresMaxConnections() string {
+	maxConnections, err := p.ConfigStore.GetItemValue(ConfigImageKey)
+	if err != nil {
+		if _, ok := err.(configstore.ErrItemNotFound); !ok {
+			// Just logged
+			p.Log.V(5).Error(err, "get database image error", "image-key", ConfigImageKey)
```

Comment:
get database image error? you're trying to get the max connection from the configstore